research/object_detection/dataset_tools/create_ava_tf_record.py

```diff
@@ -0,0 +1,457 @@
+# Copyright 2019 The MediaPipe Authors.
```

Comment:
Can you add standard license uses in OD API files ?

Reply:
Sure